addons/website/static/src/js/menu/navbar.js

```diff
@@ -1,24 +1,24 @@
 odoo.define('website.navbar', function (require) {
 'use strict';
 
-var rootWidget = require('web_editor.root_widget');
+var wWidget = require('portal.widget');
 var concurrency = require('web.concurrency');
 var Widget = require('web.Widget');
 var websiteRootData = require('website.WebsiteRoot');
 
-var websiteNavbarRegistry = new rootWidget.RootWidgetRegistry();
+var websiteNavbarRegistry = new wWidget.RootWidgetRegistry();
 
-var WebsiteNavbar = rootWidget.RootWidget.extend({
-    events: _.extend({}, rootWidget.RootWidget.prototype.events || {}, {
+var WebsiteNavbar = wWidget.RootWidget.extend({
+    events: _.extend({}, wWidget.RootWidget.prototype.events || {}, {
         'click [data-action]': '_onActionMenuClick',
         'mouseover > ul > li.dropdown:not(.show)': '_onMenuHovered',
         'click .o_mobile_menu_toggle': '_onMobileMenuToggleClick',
     }),
-    custom_events: _.extend({}, rootWidget.RootWidget.prototype.custom_events || {}, {
-        action_demand: '_onActionDemand',
-        edit_mode: '_onEditMode',
-        readonly_mode: '_onReadonlyMode',
-        ready_to_save: '_onSave',
+    custom_events: _.extend({}, wWidget.RootWidget.prototype.custom_events || {}, {
+        'action_demand': '_onActionDemand',
```

Comment:
?

Reply:
As those are event names which are even snake_cased, I think it is clearer that way. I am pretty sure you would not write `click` unquoted in the `events` object :wink:

Reply:
I also do the same for keys which are python variable names :wink: Again because they are snake_cased and it is clearer they are names that will be sent to the server but also to avoid new employees to camelCase them without caring they were python variables :)